trompace/mutations/itemlist.py

```diff
@@ -0,0 +1,197 @@
+def mutation_create_itemlist(contributor: str, name: str = None,
+                             itemlistorder: ItemListOrderType =
+                             ItemListOrderType.unordered,
+                             description: str = None):
+    """Returns a mutation for creating an itemlist object.
+
+    Arguments:
+        {itemlist_args}
+
+    Returns:
+        The string for the mutation for creating the itemlist.
+    """
+    if not isinstance(itemlistorder, ItemListOrderType):
+        raise trompace.exceptions.InvalidItemListOrderTyException(
+                                                            itemlistorder)
+
+    args = {
+        "contributor": contributor,
+        "name": name,
+        "description": description,
+        "itemListOrder": StringConstant(itemlistorder)
+    }
+
+    args = filter_none_args(args)
+
+    return format_mutation("CreateItemList", args)
+
+
+@docstring_interpolate("itemlist_args", ITEMLIST_ARGS_DOCS)
+def mutation_update_itemlist(identifier: str, contributor: str,
+                             name: str = None,
+                             itemlistorder: ItemListOrderType =
+                             ItemListOrderType.unordered,
+                             description: str = None):
+    """Returns a mutation for updating an itemlist object.
+
+    Arguments:
+        identifier: The identifier of the itemlist in the CE to be updated.
+        {itemlist_args}
+
+    Returns:
+        The string for the mutation for updating the itemlist.
+    """
+    if not isinstance(itemlistorder, ItemListOrderType):
+        raise trompace.exceptions.InvalidItemListOrderTyException(
+                                                                itemlistorder)
+
+    args = {
+        "identifier": identifier,
+        "contributor": contributor,
+        "name": name,
+        "description": description,
+        "itemListOrder": StringConstant(itemlistorder)
+    }
+
+    args = filter_none_args(args)
+
+    return format_mutation("UpdateItemList", args)
+
+
+def mutation_delete_itemlist(identifier: str):
+    """Returns a mutation for deleting an itemlist object based
+    on the identifier.
+
+    Arguments:
+        identifier: The unique identifier of the itemlist object.
+
+    Returns:
+        The string for the mutation for deleting the itemlist object
+        based on the identifier.
+    """
+
+    return format_mutation("DeleteItemList", {"identifier": identifier})
+
+
+LISTITEM_ARGS_DOCS = """name: The name of the ItemList object.
+        contributor: A person, an organization, or a service responsible
+        for contributing the ItemList to the web resource.
+        This can be either a name or a base URL.
+        description: The description of the ItemList object
+        """
+
+
+@docstring_interpolate("listitem_args", LISTITEM_ARGS_DOCS)
+def mutation_create_listitem(contributor: str, name: str = None,
+                             description: str = None):
```

Comment:
did we discuss that we would add position here too? I think it's a simple addition to put in.